neqo-qpack/src/table.rs

```diff
@@ -57,63 +58,67 @@ impl DynamicTableEntry {
 }
 
 #[derive(Debug)]
-pub struct HeaderTable {
-    qpack_side: QPackSide,
+pub(crate) struct HeaderTable {
```

Comment:
Is this change necessary?  That is, is there anything in this module that needs to be used outside of the crate?

Reply:
No, there is nothing that uses this outside crate, that is why I limited its scope.